bin/create_course.sh

```diff
@@ -222,6 +222,7 @@ create_and_set  u=rwx,g=rwxs,o=   $instructor  $ta_www_group   $course_dir/custo
 #               drwxr-s---       $HWPHP_USER        ta_www_group    uploads/bulk_pdf/
 #               drwxr-s---       $HWCGI_USER        ta_www_group    uploads/split_pdf/
 create_and_set  u=rwx,g=rxs,o=   $HWPHP_USER        $ta_www_group   $course_dir/submissions
+create_and_set  u=rwx,g=rxs,o=   $HWPHP_USER        $ta_www_group   $course_dir/forum_attachments
```

Comment:
Is there any reason to have execute allowed on these attachments?

Reply:
I believe that relates to the folder that the attachments are stored in and not the attachments themselves. The attachments don't have execution allowed, just read or write. 

![screenshot from 2018-01-17 13-18-20](https://user-images.githubusercontent.com/16356240/35059402-03a93212-fb89-11e7-876c-1781ff26b4c0.png)